### DISCUSSION TITLE
How to run Loras

### DISCUSSION THREAD

**Discussion by pavancshekar-dev** [oobabooga/text-generation-webui#6364] [Q&A]
I have couple of models in HuggingFace, and when downloading from UI it gets stored in loras/ folder automatically . How to run Lora models using Textgen UI? Can anyone please help?

**Comment by Katehuuh** [comment]
> run Lora models using Textgen UI?

In the `Model` tab, you have `LoRA(s)` which includes a dropdown. Once selected, click `Apply LoRAs`. Ensure that the LoRA was trained on the same base model.

**Comment by pavancshekar-dev** [comment]
It worked.
Thanks a lot!
